.env.testnet

```diff
@@ -34,6 +34,9 @@ L2_AIRDROP_OWNER_ADDRESS=0x84277C9255b3B6904aCfe12C2c45c6a36E81B059
 # Owner of L2VestingWallet contract
 L2_VESTING_WALLET_OWNER_ADDRESS=0x84277C9255b3B6904aCfe12C2c45c6a36E81B059
 
+# Airdrop wallet address where LSK tokens are transferred to after airdrop period ends
+L2_AIRDROP_WALLET_ADDRESS=0xaF3f4B5033Ef666b3F3EA91a0f360541cC21e5d7
```

Comment:
where does this one come from?

Reply:
From here:
https://github.com/LiskHQ/lisk-contracts/blob/main/script/data/testnet/accounts_1.json
15M LSK was sent to Airdrop wallet